wallet/src/de/schildbach/wallet/ui/dashpay/ContactsFragment.kt

```diff
@@ -139,10 +139,6 @@ class ContactsFragment : BottomNavFragment(R.layout.fragment_contacts_root), Tex
         }
         searchContacts()
 
-        invite_friend_hint.setOnClickListener {
-            InviteFriendActivity.startOrError(requireActivity())
-        }
-
```

Comment:
This removes the listener for the invite Button on the Contacts Screen (when there are no contacts or contact requests)